example/config.ini

```diff
@@ -2,6 +2,7 @@
 dbpath = /tmp/liveprofiler/
 
 [collector]
+interval = 2
```

Comment:
Assuming seconds, a comment should probably be added here saying that for production 60s is recommended.